packages/manager/apps/web/client/app/hosting/multisite/MULTISITE.html

```diff
@@ -181,8 +181,8 @@
                                         class="oui-badge"
                                         data-ng-bind="(('hosting_tab_DOMAINS_table_firewall_state_' + domain.firewall | translate) == 'hosting_tab_DOMAINS_table_firewall_state_' + domain.firewall)? domain.firewall: ('hosting_tab_DOMAINS_table_firewall_state_' + domain.firewall | translate)"
                                         data-ng-class="{
-                                            'oui-status_success': domain.firewall === 'ACTIVE',
-                                            'oui-status_error': domain.firewall === 'NONE'
+                                            'oui-badge_success': domain.firewall === 'ACTIVE',
```

Comment:
This looks like it could have been handled in the updater but this changes is applied by `
refactor: apply complex changes of migration` (https://github.com/ovh/manager/pull/3190/commits/dba28c23efd1badd8a8acee7e85a0a8190e663f0), is it a mistake ?

Reply:
No, it's sadly not handled by the updater. That's why it's done in another commit